src/sortStudents.ts

```diff
@@ -1,16 +1,51 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function findAverageGrade(gradeArr: number []): number {
+  return gradeArr
+    .reduce((accum: number, curr: number) => accum + curr, 0)
```

Comment:
```suggestion
    .reduce((accum, curr) => accum + curr, 0)
```
TypeScript understands it itself, because of gradeArr is of number[] type.

Reply:
it was marked as a mistake, that's why i put types there